Source/WebCore/platform/graphics/skia/PlatformDisplaySkia.cpp

```diff
@@ -66,6 +66,33 @@ static sk_sp<const GrGLInterface> skiaGLInterface()
 
 static thread_local RefPtr<SkiaGLContext> s_skiaGLContext;
 
+int initializeMSAASampleCount(GrDirectContext* grContext)
+{
+#if PLATFORM(GTK)
+    // GTK is desktop oriented, where using the GPU for antialiasing is not that necessary.
+    return 0;
```

Comment:
I think we should still check if WEBKIT_MSAA_SAMPLE_COUNT (see next comment) is set != 0, and if so use it. If unset return zero.